README.adoc

```diff
@@ -1,122 +1,353 @@
+
+_templates/liquid/::
+Here we store most of our prebuilding templates.
+These are _not_ Jekyll theming templates.
+These are the ones we use for generating new YAML and AsciiDoc source files from other source files and external data.
+
+content/::
+The first of our publishable directories, `content/` is the base path for documentarians' main work area.
+Everything inside the `content/` directory will be copied into the `_build/` directory early in the build process.
+
+content/assets/:::
+For content assets, rather than theming assets.
+If it illustrates your product, it probably goes here.
+If it brands your company, it probably goes in `theme/assets/`.
+
+content/pages/:::
+For AsciiDoc files of the _page_ content type.
+
+content/snippets/:::
+For content _snippets_.
+
+content/topics/:::
+For AsciiDoc files of the core _topic_ content type.
+
+data/::
+All YAML small-data files that contain content-relevant information go here.
+These data files differ from those that belong in `_configs/` (or `_ops/`) in important ways, essentially revolving around whether the data needs to be available for display.
+If it is not establishing settings or used to inform non-build functions (like in `_ops/`), the data file probably belongs in `data/`.
+Let's look at some key data files standard to LDCMF.
+
+data/meta.yml:::
+For general information about your company, URL and path info.
+This file usually contains just simple data: a big (or small) column of basic key-value pairs to create simple variables.
+
+data/products.yrml:::
+For subdivided information about your products in distinct blocks.
+Each block can be called for selective ingest during build routines using the colon signifier, such as by calling `data/products.yml:product-1`, where `product-1:` is a top-level block in the `products.yml` file.
+
+data/guides.yml:::
+This block is for content-oriented data that is distinct between the different portals or guides you're producing.
+This is often redundant to your `products.yml` file, if product editions themselves are the major point of divergence in your docs, and it is formatted the same way.
+For _this project_ (LDCMF Guides), the _guides_ are oriented toward _audiences_ (documentarians, admins, and developers), but the products (LiquiDoc and LDCMF) are distinct from this and actually documented/instructed _together_ in each guide.
++
+[TIP]
+Favoring the filename `products.yml` is conventional when products and guides (portals) have a 1:1 relationship and `guides.yml` file is superfluous.
+
+data/schema.yml:::
+Can also be `data/manifest.yml`, this crucial file provides, central manifest of all page-forming content items (pages, topics) and how they are organized (metadata such as categories into which content items fall).
+The schema file carries essential build info that lets us see relationships between topics and build content-exclusive portals from otherwise fairly dumb, decontextualized repositories.
```

Comment:
```suggestion
The schema file carries essential build info that lets us see relationships between topics and build content-exclusive portals from otherwise-decontextualized repositories.
```